### PR TITLE
PE 578 Add community tip to bundle transaction

### PR DESCRIPTION
**Pull request by jdaev**

### CODE REVIEW

File: lib/blocs/upload/upload_cubit.dart [hunk 1]
```diff
@@ -284,10 +284,19 @@ class UploadCubit extends Cubit<UploadState> {
...
+        bundleTx
+          ..addTag(TipType.tagName, TipType.dataUpload)
+          ..setTarget(await _pst.getWeightedPstHolder())
+          ..setQuantity(bundleTip);
+
         _multiFileUploadHandles.add(
-          MultiFileUploadHandle(
-              await _arweave.prepareDataBundleTx(dataBundle, profile.wallet),
-              uploadSize,
+          MultiFileUploadHandle(bundleTx, uploadSize,
```
Comment (arielmelendez) [inline lib/blocs/upload/upload_cubit.dart:299]: Is this how the code formatter decided to set up the parameters? Each on a newline might be preferable.

**Review by arielmelendez — APPROVED** [review]

File: lib/blocs/upload/upload_cubit.dart (hunk 1 above)
Comment (jdaev) [reply lib/blocs/upload/upload_cubit.dart]: [Updated](https://github.com/ardriveapp/ardrive-web/pull/297/commits/b5b7df63f2a9b8b5c4d7c990a5fa8f8e3eb5a3a4)

**Review by matibat — APPROVED** [review]
LGTM

File: lib/blocs/upload/upload_cubit.dart [hunk 2]
```diff
@@ -172,7 +172,12 @@ class UploadCubit extends Cubit<UploadState> {
       await feeTx!.sign(profile.wallet);
     }
 
-    final totalCost = uploadCost + pstFee + bundlePstFee;
+    if (_fileUploadHandles.isNotEmpty) {
+      final minimumPstTip = BigInt.from(10000000);
```
Comment (arielmelendez) [inline lib/blocs/upload/upload_cubit.dart:176]: This should be coming from a constant somewhere.

**Review by arielmelendez — APPROVED** [review]

**Pull request merged**